pet-clinic-web/src/test/java/abel/springframework/sfgpetclinic/controllers/OwnerControllerTest.java

```diff
@@ -40,23 +38,22 @@
     private static final String TELEPHONE_2 = "6725286";
     private static final String FIRST_NAME_2 = "Al";
     private static final String LAST_NAME_2 = "Berto";
-    Owner OWNER = new Owner.Builder() //TODO: Implement Builder pattern
+    Owner OWNER = new Owner.Builder()
             .withAddress(ADDRESS)
             .withCity(CITY)
             .withPets(Collections.emptySet())
             .withTelephone(TELEPHONE)
             .withFirstName(FIRST_NAME)
             .withLastName(LAST_NAME)
             .build();
```

Comment:
Can't you use the `withId ` method in Owner.Builder to set the ID here? That would allow you to get rid of the use of setters [here](https://github.com/AbelAlejandro/sfg-petclinic/blob/267b29193f8fdde1d5686554a6d50ac612b6cd64/pet-clinic-web/src/test/java/abel/springframework/sfgpetclinic/controllers/OwnerControllerTest.java#L71)